src/release/bin/shutdown.bat

```diff
@@ -24,7 +30,22 @@ if not exist "%GEOSERVER_HOME%\bin\startup.bat" goto badHome
 rem No errors
 goto shutdown
 
-
+:trySystemJava
+  echo The JAVA_HOME environment variable is not defined, trying to use System Java

```

Comment:
Is it worth checking JRE_HOME in addition to JAVA_HOME - geoserver 2 no longer requires a compiler for JSP pages and can run with a JRE.

Reply:
Hmmm, when JAVA_HOME is not defined the batch will search for any Java contained in the PATH environment variable and then find the JRE-path anyway (unless the JRE is not added to the PATH).

Also, JRE_HOME is not set by the Oracle Windows JRE  Installer, so it's normally not set on Windows platforms (as far as I have seen).